File: Stage6/Java/Problem2941.java
package Stage6.Java;

import java.util.Scanner;

public class Problem2941 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        String word = sc.nextLine();
        String[] croatia = {"c=", "c-", "dz=", "d-", "lj", "nj", "s=", "z="};

        for (String str : croatia) {
            word = word.replace(str, "*");
        }

        System.out.println(word.length());
        sc.close();
    }
}

File: Stage6/Python/Problem2941.py
def main():
    croatia = ["c=", "c-", "dz=", "d-", "lj", "nj", "s=", "z="]
    word = input()
    
    for str in croatia:
        word = word.replace(str, "*")
    
    print(len(word))
    
if __name__ == "__main__":
    main()